dockers/texcmp/texcmp.js

```diff
@@ -16,18 +15,12 @@ var readFile = Q.denodeify(fs.readFile);
 var writeFile = Q.denodeify(fs.writeFile);
 var mkdir = Q.denodeify(fs.mkdir);
 
-// ignore some tests, since they contain commands not supported by LaTeX
-var blacklist = {
```

Comment:
@kevinb7 I assume taht we should *not* put extraneous stuff in the query, right? I can make `ss_data.js` just include selected keys into its `query` property, that's easy enough. Will do so in my next commit, soon.